gematria/datasets/llvm-exegesis_warpper.S

```diff
@@ -0,0 +1,32 @@
+# LLVM-EXEGESIS-DEFREG RAX 10000
+# LLVM-EXEGESIS-DEFREG RBX 10000
+# LLVM-EXEGESIS-DEFREG RCX 10000
+# LLVM-EXEGESIS-DEFREG RDX 10000
+# LLVM-EXEGESIS-DEFREG RSI 10000
```

Comment:
It looks like you're omitting `RSP` here due to the issues that we discussed over slack? It should be preserved and set properly by `llvm-exegesis`, so it's currently a bug (thinking about it more now, some of the register values are loaded in from the stack and there's no register saving there, so it's most likely getting clobbered). I'll try and put up a patch soonish. Can you add a note/comment somewhere that it's not being set currently due to that reason?